### PR TITLE
Syndie-ship ghost roles 

### PR DESCRIPTION
**Pull request by Karmic-Skink**
## About The Pull Request
Adds some ghost roles for Mr Sergei's syndicate away mission ships.

## Why It's Good For The Game
Using custom spawners allows us to change specific objects for our use without editing what is already existing.

## Changelog
:cl:
add: Syndie Soldier and Pilot antag ghost roles
add: NT Prisoner protag ghost role
add: custom outfits for the three roles
/:cl:

### CODE REVIEW

**Comment by Karmic-Skink** [comment]
I'll need you to fill in the lore and text at some point.

Also confirm the loadouts.

File: sephora/game/objects/custom_ghost_role_spawners.dm [hunk 1]
```diff
@@ -0,0 +1,56 @@
+/obj/effect/mob_spawn/human/sephora_syndicate_crew/
```
Comment (francinum) [inline sephora/game/objects/custom_ghost_role_spawners.dm]: Make these a subtype of `/obj/effect/mob_spawn/human/sephora/`, they don't need some awkward prefix which breaks with convention.

Comment (Karmic-Skink) [reply sephora/game/objects/custom_ghost_role_spawners.dm]: Okay I've changed it.

It has created a spawner of the subtype above that just dumps a dead body on the ground, however.

**Comment by Vasily2013** [comment]
LOREBOX COMIN'

Syndie crew

`flavour_text = "<b>You are a <b><span class='big bold'>Syndicate Soldier</span><b>, part of the crew of a Syndicate Space Vessel. Your objective is to protect your ship and its valuable assets such as prisoners or navigational data at all costs. Failure will not be accepted.</b>"`

`Desc = “Although this stasis pod looks medicinal, it seems as though it's meant to preserve something for a very long time. It has a name and Syndicate flag on it.”`

Syndie pilot

`flavour_text = "<b>You are a <b><span class='big bold'>Syndicate Soldier</span><b>, part of the crew of a Syndicate Space Vessel. Your objective is to fly and protect the bridge of the ship that you have been stationed on. Keep the navigational data from falling into enemy hands.</b>"`

`Desc = “Although this stasis pod looks medicinal, it seems as though it’s meant to preserve something for a very long time. It has an Officer’s name and a Syndicate flag on it.”`

Nanotrasen Prisoner

`flavour_text = "<b>You are a <b><span class='big bold'>prisoner of the syndicate</span><b>. You are a Nanotrasen employee captured by the syndicate. Your objective is to escape the ship you are on alive. Hopefully your extensive training will help you! THIS IS NOT AN ANTAGONIST ROLE!</b>"`

`Desc = “Although this stasis pod looks medicinal, it seems as though it’s meant to preserve something for a very long time. It has a prisoner’s identification number and a Nanotrasen flag on it. It seems like it’s been bashed around quite a bit.”`

File: sephora/game/objects/custom_ghost_role_spawners.dm (hunk 1 above)
Comment (francinum) [reply sephora/game/objects/custom_ghost_role_spawners.dm]: That's fine. It's not going to be used in proper gameplay and is merely a sorting tool.

**Pull request merged**